src/main/kotlin/com/terraformation/backend/accelerator/db/ParticipantProjectSpeciesStore.kt

```diff
@@ -89,6 +106,28 @@ class ParticipantProjectSpeciesStore(
         .execute()
   }
 
+  fun fetchLastUpdatedSpeciesTime(projectId: ProjectId): Instant {
+    requirePermissions { readProject(projectId) }
+
+    val lastUpdatedTime =
+        with(PARTICIPANT_PROJECT_SPECIES) {
+          dslContext
+              .select(ID, MODIFIED_TIME)
+              .from(this)
+              .where(PROJECT_ID.eq(projectId))
+              .orderBy(MODIFIED_TIME.desc())
+              .limit(1)
+              .fetch()
+              .map { it[MODIFIED_TIME]!! }
```

Comment:
Alternately:
```kotlin
dslContext
    .select(DSL.max(MODIFIED_TIME))
    .from(this)
    .where(PROJECT_ID.eq(projectId))
    .fetchOne(MODIFIED_TIME)
```

Reply:
For some reason, when running tests I now get:
```
java.lang.IllegalArgumentException: Field ("accelerator"."participant_project_species"."modified_time") is not contained in Row (max("accelerator"."participant_project_species"."modified_time"))
```

I can see the `modified_time` being inserted correctly when I debug the test though...

Reply:
Ah, had to update the fetch to `.fetchOne(DSL.max(MODIFIED_TIME))`, works like a charm, thanks.